docs/index.rst

```diff
@@ -36,6 +36,9 @@ The asset rules are an array of JSON objects which define the asset name to whic
 
   - **rename**: Change the name of the asset. In this case a third property is included in the rule object, "new_asset_name"
 
+  - **remove**: This action will be passed a datapoint name as an argument or a datapoint type. A datapoint with that name will be removed from the asset as it passed through the asset filter. If a type is passed then all data points of that type will be removed.
+Valid data types supported are "FLOAT", "INTEGER", "STRING", "FLOAT_ARRAY", "DP_DICT", "DP_LIST", "IMAGE", "DATABUFFER", "2D_FLOAT_ARRAY", "NUMBER", "NON-NUMERIC", "FLOATING", "BUFFER". "FLOATING" maps to "FLOAT", "BUFFER" maps to "DATABUFFER". "NUMBER" type which removes both integer and floating point values and the "NON-NUMERIC" that removes everything except integer and floating point values, USER_ARRAY will remove both FLOAT_ARRAY and 2D_FLOAT_ARRAY datapoints. Nested will remove DP_DICT, ARRAY will remove FLOAT_ARRAY, 2D_ARRAY will remove 2D_FLOAT_ARRAY datapoints. Datapoint types are case insensitive.
```

Comment:
This list is not right, please see my earlier comment about what we present to the user versus what we do internally.

Reply:
fixed.